src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -571,19 +560,36 @@ llvm_verify_x86_common (Some (llvmModule :: LLVMModule x)) path nm globsyms chec
 
       let execFeatures = simpleLoopFixpointFeature ++ psatf
 
-      liftIO $ C.executeCrucible execFeatures initial >>= \case
-        C.FinishedResult{} -> pure ()
+      finalState <- liftIO $ C.executeCrucible execFeatures initial >>= \case
+        C.FinishedResult _ pr -> do
+          gp <- getGlobalPair opts pr
+          mem' <- maybe
+            (fail "internal error: LLVM Memory global not found")
```

Comment:
Perhaps this should `panic` instead of `fail`ing? I can't foresee the call to `lookupGlobal mvar (gp ^. gpGlobals)` failing unless the code itself is wrong.